### ISSUE TITLE
Clause wrapper logic / transforms

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/cicero#82]
- [ ] Wrapper Logic syntax
- [ ] Bindings clause vars to global contract vars

**Comment by jeromesimeon** [comment]
I don't understand this issue. (And yes, I have only myself to blame). @mttrbrts any idea what this is about?

**Comment by mttrbrts** [comment]
Yes, it's related to referencing data in a parent contract from an embedded clause.

For example, if my contract contains a party name in a preamble, that I want to use when I emit an obligation in a smart clause, how do I access the data?

Having clause specific data transformations is one way to solve this potentially?

This is important to making smart clauses context independent
